prow_config.yaml

```diff
@@ -45,12 +46,9 @@ workflows:
       - hack/*
       - sdk/python/v1beta1/*
       - test/e2e/v1beta1/*
-      - test/scripts/v1beta1/*
-      - test/suggestion/v1beta1/*
-      - test/workflows/*
       - manifests/v1beta1/*
       - scripts/v1beta1/*
       - prow_config.yaml
       - Makefile
-    params:
+    kwargs:
```

Comment:
This got changed?

Reply:
@johnugeorge Check here: https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/run_e2e_workflow.py#L12-L17.
It uses `kwargs` in Python scripts.